frontend-project/src/services/swagger.ts

```diff
@@ -0,0 +1,196 @@
+// This file is generated, do not manually edit this file
+// Update and use '@/scripts/generate_interface.js' instead
+
+export interface AdministrativeUnit {
+  id: string;
+  parent: string;
```

Comment:
To prawdopodobnie powinno być `nullable`, ale mam wątpliwości czy w taki sposób należy to wyrazić w TS.

Reply:
nullable to union type 
```typescript
parent: string | null
```

możliwy undefined:
```typescript
parent?: string
```

Reply:
@ad-m ewentualnie <del>mogę napisać jakiś typ generyczny</del> można użyć jakiegoś gotowego typu generycznego, wtedy składnia wyglądałaby np. tak:
```
parent: Nullable<string> 
```